commit 6b782dadcdd5bd7e15f382b0ace56170525d7145 (refs/changes/75/4175/3)
Author: Robert Mustacchi <rm@joyent.com>
Date:   2018-06-10T16:44:03+00:00 (1 year, 4 months ago)
    
    OS-7007 qede_ring_tx is too noisy
    Reviewed by: Jason King <jason.king@joyent.com>
    Approved by: John Levon <john.levon@joyent.com>

diff --git a/usr/src/uts/common/io/qede/qede_fp.c b/usr/src/uts/common/io/qede/qede_fp.c
index 7d415cbb2a..8018dac365 100644
--- a/usr/src/uts/common/io/qede/qede_fp.c
+++ b/usr/src/uts/common/io/qede/qede_fp.c
@@ -1772,8 +1772,6 @@ qede_ring_tx(void *arg, mblk_t *mp)
 	}
 
 	if (!qede->params.link_state){
-		qede_print_err("!%s(%d): Link !up for xmit",
-		    __func__, qede->instance);
 		goto exit;
 	}
 
